packages/bootstrap-4/src/SelectWidget/SelectWidget.tsx

```diff
@@ -82,7 +82,7 @@ const SelectWidget = ({
       </Form.Label>
       <Form.Control
         as="select"
-        custom
+        bsPrefix="custom-select"
```

Comment:
When I just removed `custom`, the tests failed because the `className` changed from `custom-select` to `form-control`. The `bsPrefix` prop seems to be the way to approach this, based on the docs at https://react-bootstrap.github.io/forms/form-control/#form-control-props

Reply:
What problem is this solving? Is there an issue associated with the change?

Reply:
Yes, I put it in the description. This fixes #2648